### PR TITLE
[FIX] fix test schedule db backup.

### PR DESCRIPTION
**Pull request by kzvyahin**
Change locator. Change wait mechanizm for db backup schedule

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 0f6cf8ad6ddd9a73dba2c16037f1611d0bb887a9:

`cfme/tests/configure/test_db_backup_schedule.py`:
- :red_circle: Line 7:1: **F401** *'InfoBlock' imported but unused*

Please, rectify these issues :smirk: .
*CFME QE Bot*

File: cfme/configure/configuration/__init__.py [hunk 1]
```diff
@@ -1445,7 +1445,7 @@ class DatabaseBackupSchedule(Schedule):
         ("time_zone", {
             version.LOWEST: Select("select#time_zone"),
             '5.5': AngularSelect('time_zone')}),
-        ("start_date", Calendar("miq_angular_date_1")),
+        ("start_date", Calendar("start_date")),
```
Comment (jkrocil) [inline cfme/configure/configuration/__init__.py]: We want to use version pick here - `miq_angular_date_1` for 5.4 and `start_date` for 5.5+.

**Comment by dajoRH** [comment]
Lint report for commit 936b098ac293fbcc4e36e943d5208ded66f07aec:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/configure/configuration/__init__.py [hunk 2]
```diff
@@ -1555,6 +1557,12 @@ def update(self, updates, cancel=False, samba_validate=False):
         else:
             form_buttons.save()
 
+    @property
+    def last_date(self):
```
Comment (jkrocil) [inline cfme/configure/configuration/__init__.py]: We need to make sure, that we are on the right page, when trying to fetch data from the table.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e7459db275ea39c3687bd4745a68be5b2d9b709a:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ec3f96a38265cfa95bddf3d214c2b32f06015dab:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by jkrocil** [comment]
Fail on upstream is unrelated. Merging.

**Pull request merged**